.pre-commit-config.yaml

```diff
@@ -39,6 +39,18 @@ repos:
       - id: flake8
         args: ["--select=E9,F63,F7,F82"]
 
+  - repo: https://github.com/returntocorp/semgrep
+    rev: 'v0.12.0'
+    hooks:
+      - id: semgrep
+        args: ['--config', 'https://semgrep.live/p/python', '--include', '*.py', '--precommit', '--error']
```

Comment:
Trying this with something like `/p/r2c` was really slow and produced lots of parse errors. Using `/p/python` and `--include *.py` sped things up.